doc/source/whatsnew/v0.24.0.rst

```diff
@@ -280,6 +280,7 @@ Other Enhancements
 - :meth:`round`, :meth:`ceil`, and meth:`floor` for :class:`DatetimeIndex` and :class:`Timestamp` now support a ``nonexistent`` argument for handling datetimes that are rounded to nonexistent times. See :ref:`timeseries.timezone_nonexistent` (:issue:`22647`)
 - :class:`Resampler` now is iterable like :class:`GroupBy` (:issue:`15314`).
 - :meth:`Series.resample` and :meth:`DataFrame.resample` have gained the :meth:`Resampler.quantile` (:issue:`15023`).
+- :meth:`TimeGrouper._get_period_bins` now accounts for the base argument passed through the :class:`PeriodIndexResampler` (:issue:`23882`)
```

Comment:
We generally don't reference private methods in our `whatsnew` entries.  Is it possible to describe the fix without referencing it?  A good way to come up with a description is look at your test and see what new behavior is fixed based on what you wrote.